src/components/common/NameRequestInfo.vue

```diff
@@ -168,7 +169,7 @@ export default class NameRequestInfo extends Mixins(DateMixin) {
 
   // Global Store
   @State(state => state.stateModel.nameTranslations)
-  readonly nameTranslations!: Array<string>
+  readonly nameTranslations!: NameTranslationIF[]
```

Comment:
I know this is existing code, but this variable from the store is exactly the same as getNameTranslations below.
```
export const getNameTranslations = (state: any): Array<string> => {
  return state.stateModel.nameTranslations
}
```

Could you please replace this state variable with the getter below in this file?